launch-dedicated.bat

```diff
@@ -0,0 +1,17 @@
+@echo on
+
+set Name="Dedicated Server"

```

Comment:
Yeah it will. Test this with Game.Mod:
```
openra.game Game.Mod="ra blah"
```
This takes you to the mod chooser because there's no mod called `"ra`
```
openra.game Game.Mod=ra blah
```
This takes you to the ra main menu because blah was chopped off. It's interpreting it as `Game.Mod=ra` and `blah` as a separate argument entirely
The arguments parser is dumb and doesn't know how to parse things with quotes/spaces correctly.